rules/integrations/aws/defense_evasion_config_service_rule_deletion.toml

```
@@ -23,17 +23,72 @@ index = ["filebeat-*", "logs-aws*"]
 interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
-name = "AWS Config Service Tampering"
-note = """## Config
+name = "AWS Config Resource Deletion"
+note = """## Triage and analysis
+
+### Investigating AWS Config Resource Deletion
+
+AWS Config provides a detailed view of the configuration of AWS resources in your AWS account. This includes how the
+resources are related to one another and how they were configured in the past so that you can see how the configurations
+and relationships change over time.
+
+This rule looks for the deletion of AWS Config resources using various API actions. Attackers can do this to cover their
+tracks and impact security monitoring that relies on these sources.
+
+#### Possible investigation steps
+
+- Identify the user account that performed the action and whether it should perform this kind of action.
+- Identify the AWS resource that was involved and its role in the environment, criticality, and ownership. Also investigate
```

This list is a little awkward because all the items belong to "its" but could be misread another way; reordering might help:
```suggestion
- Identify the AWS resource that was involved and its criticality, ownership, and role in the environment. Also investigate
```